exercises/rest-api/canonical-data.json

```diff
@@ -396,18 +396,18 @@
                                 "name": "Adam",
                                 "owes": {},
                                 "owed_by": {},
-                                "balance": 0
+                                "balance": 0.0
                             },
                             {
                                 "name": "Bob",
                                 "owes": {},
                                 "owed_by": {},
-                                "balance": 0
+                                "balance": 0.0
                             }
                         ]
                     }
                 }
             ]
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
A small nit pick, but this change is not related to the other change you made earlier in the file.

Reply:
I've created an issue for this, as it is a common issue with PR's created using the GitHub interface: https://github.com/exercism/problem-specifications/issues/1443